docs/Users_Guide/release-notes.rst

```diff
@@ -76,6 +76,351 @@ METplus Version 6.0.0 Beta 1 Release Notes (2023-09-15)
+
+    climo_cdf = {                           climo_cdf = {
+       cdf_bins    = 2;                |       cdf_bins    = 1;
+       center_bins = FALSE;                    center_bins = FALSE;
+       write_bins  = TRUE;                     write_bins  = TRUE;
+    }                                       }
+
+The :ref:`GridStat - MET Configuration<grid-stat-met-conf-climo-cdf>` section of
+the Python Wrappers chapter shows that :term:`GRID_STAT_CLIMO_CDF_BINS` is the
+METplus config variable that sets the **climo_cdf.cdf_bins** variable
+in the GridStat wrapped MET config file.
+Note that slightly redundant :term:`GRID_STAT_CLIMO_CDF_CDF_BINS` is also
+supported to match the naming convention <WRAPPER_NAME>_<DICT_NAME>_<VAR_NAME>.
+
+To set the variable found in the user-defined wrapped MET config
+file, set the following in the METplus config file::
+
+    GRID_STAT_CLIMO_CDF_BINS = 2
+
+**Unsupported Variable Example:**
+
+::
+
+    < ci_alpha  = [ 0.3 ];
+    ---
+    > ci_alpha  = [ 0.05 ];
+
+The **ci_alpha** variable is not found in the
+:ref:`GridStat - MET Configuration<grid-stat-met-conf>` section.
+Reviewing the wrapped MET config file in this section will reveal that there is
+no environment variable that sets the variable.
+
+In this case, add the desired setting including the variable name to the
+MET config overrides variable without changing any formatting::
+
+    GRID_STAT_MET_CONFIG_OVERRIDES = ci_alpha  = [ 0.3 ];
+
+See :ref:`met-config-overrides` for more information.
+
+Variables referencing user-defined environment variables
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Some users may have set their own environment variables and referenced them in
+their wrapped MET config file. An environment variable that is not found in the
+error logs listing deprecated environment variables and does not start with
+**METPLUS_** was likely defined by the user. These variables will no longer
+be supported, so the variables that reference them should be set using METplus
+configuration variables instead.
+
+Verify results
+^^^^^^^^^^^^^^
+
+Once all of the changes are made, it is recommended to confirm that the use case
+produces the same results. A good way to confirm this is to run the use case
+using the version of METplus that was previously used, run the use case with the
+new version with use case updates, then compare the output.
+
+A diff utility is provided with the METplus wrappers that can be used to compare
+two directories that contain METplus output while filtering out differences that
+are not relevant, such as skipping log files that contain different timestamps
+and ignoring version number differences in stat output files. The diff utility
+can be found in the METplus installation location at
+**metplus/util/diff_util.py**. Call the script on the command line passing in
+the two directory (or file) paths to compare::
+
+    ./metplus/util/diff_util.py /path/to/output_one /path/to/output_two
```

Comment:
Using the "./" here may be misleading, since the instructions do not say to change directories to the METplus installation location.  If a user is not familiar with "./" and is blindly copying and pasting the path, this command may not work.

Reply:
This command contains fake paths as an example, so it will not work if blindly copy/pasted regardless of where it is run. I could change it to something similar to the above example of a path in the MET installation area:

`<METPLUS_INSTALLATION_LOCATION>/metplus/util/diff_util.py /path/to/output_one /path/to/output_two`

or

`METplus-X.Y.Z/metplus/util/diff_util.py /path/to/output_one /path/to/output_two`

or

`/path/to/METplus-X.Y.Z/metplus/util/diff_util.py /path/to/output_one /path/to/output_two`

What do you think?

Reply:
I prefer `/path/to/METplus-X.Y.Z/metplus/util/diff_util.py /path/to/output_one /path/to/output_two`.